rental.rb

```diff
@@ -0,0 +1,7 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date)
+    @date = date
+  end
```

Comment:
it would be better to add a book and a person to the rental in order to achieve `many to many relationships through rental` please try to add that it should be something like 

```ruby
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
```

Reply:
Done